File: HAI704I - Architectures Distribuées/TP2/TP2-Server/TP2-ServiceServer/model/customer/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TP2_Server.model.customer.card;

namespace TP2_Server.model.customer
{
    public class Customer
    {
        public string FirstName
        {
            get;
            set;
        }

        public string LastName
        {
            get;
            set;
        }

        public CreditCard Card
        {
            get;
            set;
        }

        public Customer() : this("", "", null)
        {
        }

        public Customer(string firstName, string lastName, CreditCard cards)
        {
            this.FirstName = firstName;
            this.LastName = lastName;
            this.Card = cards;
        }
    }
}

File: HAI704I - Architectures Distribuées/TP3/TP3-Hotel/src/main/java/m1/hotels/model/customer/Customer.java
package m1.hotels.model.customer;

import m1.hotels.model.customer.card.CreditCard;

public class Customer
{
    private String firstName;
    private String lastName;
    private CreditCard card;

    public Customer()
    {
    	this("", "", null);
    }

    public Customer(String firstName, String lastName, CreditCard cards)
    {
        this.firstName = firstName;
        this.lastName = lastName;
        this.card = cards;
    }

	public String getFirstName()
	{
		return firstName;
	}

	public void setFirstName(String firstName)
	{
		this.firstName = firstName;
	}

	public String getLastName()
	{
		return lastName;
	}

	public void setLastName(String lastName)
	{
		this.lastName = lastName;
	}

	public CreditCard getCard()
	{
		return card;
	}

	public void setCard(CreditCard card)
	{
		this.card = card;
	}
    
    
}
